### ISSUE TITLE
Sorted list of Bees

### ISSUE THREAD

**Issue by internetimagery** [muesli/beehive#246]
Hi all!

This is a really promising little application. I'm looking forward to getting stuck in with it. 

One thing that that I would like to request however is if it's possible to sort the bees in the main interface. Perhaps even statically, if that is easier.

I'm thinking, in descending order of specificity. eg:

* Harware related bees together (serial ports / filesystem watching etc).
* Generic protocols together ( email, http servers, irc, etc).
* Service based APIs. (the rest).

It hurts discoverability somewhat when first evaluating the software, in trying to work out all the capabilities on offer, when they're all strewn around as they are. I can only imagine this getting more extreme as more options are added.

From my naive perspective, a real selling point for this app, over the competition is not just in its simplicity and lightweight. But also in the generics on offer. I can run mozjpeg on files that are added to a folder, then upload them to an online service for backup automatically. I can scrape a website when it doesn't have an API that can be utilized. etc

With these generic bees being a real selling point, it makes sense to me that they're on show cleanly divided in their own section.

Thanks a lot all. Keep at it. This is looking great thus far! :)

**Comment by muesli** [comment]
Yeah, that's a great idea! I'm working on a refactor of the entire admin interface and will take that into account.
